extensions/arc/runtime/src/main/java/io/quarkus/arc/config/NativeBuildTime.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.arc.config;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * A config property injected during the static initialization phase of a native image build may result in unexpected errors
```

Comment:
```suggestion
 * Allows a configuration property injection point to retrieve values during static initialization.
 * <p>
 * A config property injected during the static initialization phase of a native image build may result in unexpected errors
```

Reply:
We should improve the wording here, as it's not only about injecting properties.